lib/upgrade/migrator/migrate.go

```diff
@@ -387,6 +387,16 @@ func (m *migrationHelper) queryAndConvertResources() (*MigrationData, error) {
 		return nil, err
 	}
 
+	if m.clientv1.IsKDD() {
```

Comment:
The fix below iterates through all resources to update just the nodes with the IPIP tunnel address.  Moving this up here just to minimize how many resources it iterates through.  Should be minor impact though and can back this out if you'd prefer.